### PR TITLE
Comp no wname in kcontrol name

### PR DESCRIPTION
**Pull request by jsarha**
Ok, this PR has evolved enough so that the branch name does not anymore match the content, so I guess its time to turn the page and create a new one. The predecessor can be found as draft here: https://github.com/thesofproject/sof/pull/7227

### CODE REVIEW

**Comment by ranj063** [comment]
@jsarha I was discussing the volume test in sof-ci with @marc-hb yesterday and we realized that the IPC4 volume tests do nothing today because there are no mixer names with "PGA" in the name. Now, if we remove the widget name in the mixer names altogether, the tests will also no nothing for IPC3 as well. So we first need to think about what the volume tests do before we move ahead with this change

File: tools/topology/topology2/include/components/eqiir.conf [hunk 1]
```diff
@@ -61,4 +61,5 @@ Class.Widget."eqiir" {
 	no_pm			"true"
 	num_input_pins		1
 	num_output_pins		1
+	no_wname_in_kcontrol_name	"true"
```
Comment (ranj063) [inline tools/topology/topology2/include/components/eqiir.conf:64]: why not set this to true in widget-common.conf?

**Comment by marc-hb** [comment]
Context:
- https://github.com/thesofproject/sof-test/issues/1068
- https://github.com/thesofproject/sof-test/issues/1069

File: tools/topology/topology2/include/components/eqiir.conf (hunk 1 above)
Comment (lgirdwood) [reply tools/topology/topology2/include/components/eqiir.conf:64]: ack, seems that base class is best place.

**Comment by jsarha** [comment]
Moved the default value change to widget-common.conf.

**Comment by jsarha** [comment]
> @jsarha I was discussing the volume test in sof-ci with @marc-hb yesterday and we realized that the IPC4 volume tests do nothing today because there are no mixer names with "PGA" in the name. Now, if we remove the widget name in the mixer names altogether, the tests will also no nothing for IPC3 as well. So we first need to think about what the volume tests do before we move ahead with this change

The simplest way would be using amixer numid notation to identify the mixers, but maybe a regex pattern to match a particular mixer would be easier to follow.

**Comment by jsarha** [comment]
> Moved the default value change to widget-common.conf.

File: tools/topology/topology2/include/components/eqiir.conf (hunk 1 above)
Comment (jsarha) [reply tools/topology/topology2/include/components/eqiir.conf]: done

**Comment by lgirdwood** [comment]
@ranj063 good for you ?

**Review by ranj063 — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]

**Pull request closed without merging**